pkg/controller/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -111,18 +112,47 @@ func TestCreateMultipleUserAccountsSuccessful(t *testing.T) {
 	memberClient := test.NewFakeClient(t)
 	memberClient2 := test.NewFakeClient(t)
 	hostClient := test.NewFakeClient(t, mur)
+	memberClient.MockGet = func(ctx context.Context, key types.NamespacedName, obj runtime.Object) error {
+		if _, ok := obj.(*routev1.Route); ok {
+			obj = &routev1.Route{
+				Spec: routev1.RouteSpec{
+					Host: "host",
+					Path: "console",
+				},
+			}
+			return nil
+		}
+		return memberClient.Client.Get(ctx, key, obj)
+	}
+	memberClient2.MockGet = func(ctx context.Context, key types.NamespacedName, obj runtime.Object) error {
+		if _, ok := obj.(*routev1.Route); ok {
+			obj = &routev1.Route{
+				Spec: routev1.RouteSpec{
+					Host: "host",
+					Path: "console",
+				},
+			}
+			return nil
+		}
+		return memberClient2.Client.Get(ctx, key, obj)
+	}
```

Comment:
Or you could do this instead:
```suggestion
	memberClient := test.NewFakeClient(t, consoleRoute(), cheRoute(false))
	memberClient2 := test.NewFakeClient(t, consoleRoute(), cheRoute(false))
	hostClient := test.NewFakeClient(t, mur)
```

Reply:
ah right, of course!

Reply:
done in 072a91f (I wasn't aware of this `consoleRoute` func!)